src/main/java/com/mrbysco/unhealthydying/config/UnhealthyConfig.java

```diff
@@ -51,7 +51,7 @@
 
 			regenHealth = builder
 					.comment("When set to true allows you to gain back health upon killing set target(s) [default: false]")
-					.define("regenHealth", false);
+					.define("regenHealth", true);
```

Comment:
Why did you change the default config value?

Reply:
Sorry, you may remove these changes